### ISSUE TITLE
Download APA tables

### ISSUE THREAD

**Issue by wibeasley** [IALSA/IALSA-2015-Portland#35]
We'd like for the user to be able to click a button to download the relevant tables in APA format.  This will help collaborators to create their own publications more quickly.

We’ll base it on this demo http://shiny.rstudio.com/gallery/download-knitr-reports.html.

This demo kinda conceals the difficulty of connecting the Shiny app with the knitr report.  They’re really separate pieces of code that hardly talk to each other.  For instance, it’s tricky getting the Shiny inputs to influence/filter the knitr report that’s downloaded.  But I think we have a plan that will do the basics for Cambridge.

**Comment by wibeasley** [comment]
@andkov are you still planning to transition to a dashboard?  I forgot if you wanted help with that or not.  I'd like for that to happen before I add the 'Save as APA Table' feature.  But if you don't think the dashboard transition will happen for a while, I'll add the table before that.

**Comment by andkov** [comment]
@wibeasley, yes. I've created the starting point for #36 and was hoping to make it functional  before tomorrow's conference at 12 central.  I have time before 7 pm central or after 10 central today (Jul 9). Please let me know if you'd have some time for a session, i'd like to give it a try
